src/applications/personalization/profile/tests/e2e/personal-and-contact-info/modals.cypress.spec.js

```diff
@@ -37,8 +37,8 @@ const checkModals = options => {
 
   // Make an edit
   cy.get(`#${editLineId}`)
-    .click()
-    .type('test');
+    .click({ force: true })
+    .type('test', { force: true });
```

Comment:
Just curious if `force` is actually needed for `type()` to work. Seems like once the click happens, typing would work.

Reply:
@erikphansen The force is indeed needed. I thought adding the `force` to the `click()`would be sufficient, but the `type()` started failing after that. Seems like there's a minor rendering race condition that causes one or the other to fail, so we need the `force` in both spots.

Reply:
Thanks for confirming, @ddzz !